desktop/src/app/components/gallery/home/gallery.html

```diff
@@ -2,36 +2,13 @@
 <bl-local-template-drop-zone *ngIf="autoStorageService.hasArmAutoStorage | async">
     <div class="header">
         <h1>{{'gallery.title' | i18n }}</h1>
-        <bl-refresh-btn [refresh]="refresh" type="round"></bl-refresh-btn>
-        <div class="flex-separator"></div>
-        <bl-local-template-pick-button></bl-local-template-pick-button>
     </div>
     <div class="gallery-content">
-        <div class="gallery-applications">
-            <div class="filter">
-                <input autofocus blInput [formControl]="quicksearch" placeholder="Search">
+        <bl-banner class="gallery-deprecation-warning" type="warning">
+            <div message>
+                Gallery applications are no longer supported as of February 29, 2024. See the Azure Batch rendering VM images & licensing retirement announcement for details.
```

Comment:
Can we extract this out to a localized string? If we switch the link text to "Learn more" we can use `common.learnMore` for that, which is already localized.

Reply:
We should also parameterize the date here. See `pool-create-basic-dialog.deprecationWarning.imageNearingEndOfLife` for an example.

Reply:
Updated